plutus-chain-index/app/Marconi/Index/Utxo.hs

```diff
@@ -98,13 +100,10 @@ open
   -> Depth
   -> IO UtxoIndex
 open dbPath (Depth k) = do
-  ix <- fromJust <$> Ix.newBoxed query store onInsert k ((k + 1) * 4) dbPath
+  ix <- fromJust <$> Ix.newBoxed query store onInsert k ((k + 1) * 2) dbPath
```

Comment:
Can you leave a note explaining this `(k + 1) * 2`?

Reply:
That argument represents the buffer used in the indexer. I am simply running the indexer and trying to adjust the arguments to get a reasonable memory usage / time compromise.